### PR TITLE
Improvements to existing Confidential Computing documentation (new)

### PR DESCRIPTION
**Pull request by 0rnela**
Current documentation refers to CoCo from the earlier development stages. This PR updates the screenshots and/or instructions.

[ ]  master
[x] 5.0 https://github.com/uyuni-project/uyuni-docs/pull/3189

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/administration/pages/confidential-computing.adoc [hunk 1]
```diff
@@ -98,30 +98,38 @@ image::attestation1.jpg[]
...
 . For the selected system, go to tab menu:Audit[Confidential Computing > List Attestations].
-. Select the report you want to view.
+. Find and select the report you want to view.
 +
-image::attestation3.jpg[]
+image::attestation3-v2.jpg[]
 +
-. In the column [literal]``Actions`` click the icon to open the report.
-  Tab [literal]``Overview`` will open.
+. After clicking on the selected attestation report tab [literal]``Overview`` will open.
```
Comment (keichwa) [inline modules/administration/pages/confidential-computing.adoc:111]:
```suggestion
. After clicking the selected attestation report tab [literal]``Overview`` will open.
```

**Review by keichwa — COMMENTED** [review]
Usually you do not need the + connector between image:: and the next list item.  Did you check that PDF and HTML builds ok?

**Comment by 0rnela** [comment]
> Usually you do not need the + connector between image:: and the next list item. Did you check that PDF and HTML builds ok?

Yes, PDFs are OK.

**Comment by 0rnela** [comment]
> Usually you do not need the + connector between image:: and the next list item. Did you check that PDF and HTML builds ok?

@keichwa I have removed extra + connectors, thank you for pointing that out.

**Pull request merged**